changelog.d/14573.misc

```diff
@@ -0,0 +1 @@
+Push complement-synapse docker images to ghcr.io repository.
```

Comment:
I renamed the existing `.docker` readme to `.misc`. I think `.docker` is limited to "the" docker image that we expect server operators to use. This is an internal testing tool only.